src/Avalonia.Animation/Animation.cs

```diff
@@ -75,6 +75,16 @@ private static Type GetAnimatorType(AvaloniaProperty property)
         /// Easing function to be used.
         /// </summary>
         public Easing Easing { get; set; } = new LinearEasing();
+ 
+        /// <summary>
+        /// Sets the speed multiple for this animation.
+        /// </summary>
+        public double SpeedRatio { get; set; } = 1d;
+
+        /// <summary>
+        /// Sets the behavior for having a delay between repeats for this animation.
+        /// </summary>
+        public bool DelayBetweenRepeats { get; set; }
```

Comment:
A delay of how long? Might be worth explaining what this means a little in a `<remarks>` section of the xaml docs.